tools/PsesLogAnalyzer/Types.ps1

```diff
@@ -0,0 +1,152 @@
+enum PsesLogLevel {
+    Diagnostic;
```

Comment:
Are the semicolons needed here?

Reply:
No. Bad habit from when I defined simple enums on a single line. I'll fix it.

Reply:
Fixed.